pkg/agent/controller/egress/egress_controller.go

```diff
@@ -153,6 +155,9 @@ type EgressController struct {
 	serviceCIDRUpdateCh  chan struct{}
 	// Declared for testing.
 	serviceCIDRUpdateRetryDelay time.Duration
+
+	meterTracker      map[uint32]*crdv1b1.RateLimit
```

Comment:
Yes, it's 0-255 using `uint8` could save some memory. I choose `uint32` because, in OVS, meterID is `uint32` and the egress mark, which we leverage its value as the meterID, is also `uint32`. It could save us from continuing to convert the type.